react/table/public/index.html

```diff
@@ -42,17 +42,15 @@
             keycloak,
           };
 
-          window.entando.keycloak
-            .init({ onLoad: 'check-sso' })
-            .success(onInit);
+          window.entando.keycloak.init({ onLoad: 'check-sso' }).success(onInit);
         }
         return initKeycloak;
       })();
     </script>
   </head>
   <body onLoad="onLoad();">
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <conference-table />
+    <conference-table pagination-mode="pagination" />
```

Comment:
can we consider one of the modes (e.g. pagination) as the default one, so the attribute could not need to be explicitly set?